languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -61343,12 +61343,13 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token>,</token>
                 </pattern>
                 <message>'Nachdem' drückt standardsprachlich Vorzeitigkeit aus und sollte daher nicht mit dem Präteritum verwendet werden. Verwenden Sie das Perfekt (Präsens im Hauptsatz) oder Plusquamperfekt (Präteritum im Hauptsatz) oder 'als' zum Ausdrücken von Gleichzeitigkeit.</message>
-                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> war</suggestion>
-                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> hatte</suggestion>
+                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> <match no="2" postag="VER:([123].*)PRT.*" postag_replace="VER:AUX:$1PRT" postag_regexp="yes">sein</match></suggestion>
+                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> <match no="2" postag="VER:([123].*)PRT.*" postag_replace="VER:AUX:$1PRT" postag_regexp="yes">haben</match></suggestion>
                 <example correction="gekommen war|gekommen hatte">Nachdem der Brief <marker>kam</marker>, ging ich nach Hause.</example>
+                <example correction="gesagt warst|gesagt hattest">Nachdem du etwas <marker>sagtest</marker>, erstarrten ihre Mienen vollkommen vor Ehrfurcht.</example>
```

Comment:
Hier müsste man vlt. überlegen, ob man komische `suggestions` in Kauf nehmen will.
Bei Sätzen wie: "Nachdem ich das wusste, habe ich es geändert." kommen dann Vorschläge wie "gewusst war"
Wie siehtst du das @danielnaber ?

Reply:
Wenn *nur* falsche Vorschläge kommen für einen Fehler ist das natürlich blöd, das sollten wir vermeiden. Ansonsten verlasse ich mich da auf dein Urteil, @St-ac-y

Reply:
@affemitkaraffe @danielnaber  ich würde die Regel so aufteilen, dass Verben mit "haben" und Verben mit "sein" getrennt werden (so viele mit "sein" gibt es ja nicht, zb. alle Verben der Bewegung) damit nicht so komische Vorschläge dabei rauskommen. Der User könnte eben denken, dass es auch so richtig sein könnte, obwohl es nur ein generischer Vorschlag ist, der nicht in allen Situationen stimmt. Was denkst du, @AgnesKleinhans ?

Reply:
@St-ac-y Ja, für diesen Ansatz benötigen wir eine Liste aller Verben, die ausschließlich mit _haben_ oder _sein_ funktionieren. Ich stimme dir zu, dass es etwas unglücklich ist, wenn wir einen Vorschlag liefern, der grammatikalisch komplett falsch ist. In der `prem4` liegt die Entität `verbenmitsein`, vielleicht hilft das weiter?